frenai/assetlist.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "frenai",
+  "assets": [
+    {
+      "description": "The native token of frenai.",
+      "denom_units": [
+        {
+          "denom": "ufren",
+          "exponent": 0
+        },
+        {
+          "denom": "fren",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "ufren",
+      "name": "frenai",
+      "display": "fren",
+      "symbol": "FREN",
+      "coingecko_id": "frenai",
+      "keywords": [
+        "ai",
+        "ai polling",
+        "rpc",
+        "ai rpc",
+        "ai grpc",
+        "ai rest api",
+        "ai chatbot",
+        "ai processed data"
+      ],
+      "socials": {
+        "website": "https://fren.ai",
+        "twitter": "https://twitter.com/frenaihub"
+      },
+      "images": {
+        "png": "https://raw.githubusercontent.com/frenailabs/chain-registry/master/frenai/images/fren.png",
```

Comment:
These URIs must point to the Cosmos Chain Registry. I think simply replacing 'frenailabs' org with 'cosmos' should do the trick.

Reply:
Replaced.